.changeset/gorgeous-swans-try.md

```diff
@@ -0,0 +1,6 @@
+---
+"@nomicfoundation/hardhat": minor
+"@nomicfoundation/hardhat-vendored": patch
```

Comment:
A couple of things:

- `@nomicfoundation/hardhat` should be `hardhat`
- The changeset for Hardhat should be patch. I don't think this merits a minor bump.
- `@nomicfoundation/hardhat-vendored` is a new package, so it shouldn't be included here. Otherwise its first version will be 3.0.1, not 3.0.0. I checked locally that `changeset publish` will publish `hardhat-vendored` even if it doesn't have a changeset, because it detects that it's not published yet. (I'm not entirely sure that the CI will have the same behavior, but I don't see why it wouldn't.)

```suggestion
"hardhat": patch
```